src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeModifiers.java

```diff
@@ -108,7 +108,8 @@ private static Tuple<GTRecipe, Integer> tryParallel(IRecipeCapabilityHolder hold
         int mid = (min + max) / 2;
 
         GTRecipe copied = original.copy(ContentModifier.multiplier(mid), modifyDuration);
-        if (!copied.matchRecipe(holder).isSuccess() || !copied.matchTickRecipe(holder).isSuccess()) {
+        if (!copied.matchRecipe(holder).isSuccess() || !copied.matchTickRecipe(holder).isSuccess() ||
+            !(RecipeHelper.getRecipeEUtTier(copied) <= ((ITieredMachine)holder).getTier())) {
```

Comment:
can you be absolutely sure it'll always be a tiered machine?

Reply:
Currently, those that can use the parallel control cabin are, but it’s better to add a judgment just in case.